first cut at spec for governance dashboard needs

levity opened the pull request.

levity committed:
first cut at gov spec

levity commented:
attn @jparklev

jparklev commented on gov.graphql:
```diff
@@ -0,0 +1,44 @@
+
+type ProposalOption {
+  id: Int
+  label: String
```
Is `label` something you're thinking we should have on-chain? Or should it be fetched from the CMS in this case?

jparklev commented on gov.graphql:
```diff
@@ -0,0 +1,44 @@
+
+type Query {
+  # when blockNumber is specified, the votes and vote balances returned should
+  # match the state at that block. when it is not specified, it should default
```
What we could do: when a vote has reached a `VOTING_ENDED` state, all child vote balances should, for all future blocks, return what they were for that proposal at it's ending block (& same for all of its ProposalOptions' `voteBalanceSum`).

levity replied:
ah, yes, that makes more sense. so the default is the latest block or the block at which the vote ended, whichever is earlier.

levity committed:
small tweaks

rename ipfsHash to documentHash, remove title,
clarify purpose of blockNumber argument

levity committed:
remove option label

levity commented:
Thx for merging. What’s the process for getting this through the
development backlog? Is there a JIRA issue we can follow keep up to date
about its status?
On Mon, Oct 22, 2018 at 12:30 AM dc <notifications@github.com> wrote:

> Merged #3 <https://github.com/makerdao/vulcan.spec/pull/3> into master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/makerdao/vulcan.spec/pull/3#event-1917069596>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAL0g5Gk9XqbhSu8WGNYaKm0swEYS61kks5unXP3gaJpZM4XsSTB>
> .
>
